src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -257,6 +257,70 @@ public function revenueBetweenDates(\DateTime $from, \DateTime $to, $state = nul
         ;
     }
 
+    public function revenueBetweenDatesGroupByDate(array $configuration=array())
+    {
+        $groupBy = '';
+        foreach ($configuration['groupBy'] as $groupByArray ) {
+            $groupBy = $groupByArray.'(date)'.' '.$groupBy;
+        }
+        $groupBy = substr($groupBy, 0, -1);
+        $groupBy = str_replace(' ', ', ', $groupBy);
+        
+        $queryBuilder = $this->getQueryBuilderBetweenDatesGroupByDate(
+            $configuration['start'],
+            $configuration['end'],
+            $groupBy);
+        
+        $queryBuilder
+            ->select('DATE(o.completed_at) as date','TRUNCATE(SUM(o.total)/ 100,2) as "total sum"')
```

Comment:
The precision is 4 as default unfortunatly. It results in prices like 123.1200. If it fits our needs i can change it.